api_tests/nodes/serializers/test_serializers.py

```diff
@@ -91,6 +92,49 @@ def test_node_serializer(self, user):
             templated_from).path == '/{}nodes/{}/'.format(API_BASE, node._id)
 
 
+@pytest.mark.django_db
+class TestSparseNodeSerializer:
+
+    def test_sparse_node_serializer(self, user):
+
+        #   test_node_serialization
+        parent = ProjectFactory(creator=user)
+        node = NodeFactory(creator=user, parent=parent)
+        req = make_drf_request_with_version(version='2.15')
+        result = SparseNodeSerializer(node, context={'request': req}).data
+        data = result['data']
+        assert data['id'] == node._id
+        assert data['type'] == 'sparse-nodes'
+
+        # Attributes
+        attributes = data['attributes']
+        assert attributes['title'] == node.title
+        assert attributes['description'] == node.description
+        assert attributes['public'] == node.is_public
+        assert set(attributes['tags']) == set(node.tags.values_list('name', flat=True))
+        assert 'current_user_can_comment' not in attributes
+        assert 'license' not in attributes
+        assert attributes['category'] == node.category
+        assert 'registration' not in attributes
+        assert attributes['fork'] == node.is_fork
+
+        # Relationships
+        relationships = data['relationships']
+        assert 'region' not in relationships
+        assert 'children' in relationships
+        assert 'detail' in relationships
+        assert 'contributors' in relationships
+        assert 'files' not in relationships
+        assert 'parent' in relationships
+        assert 'affiliated_institutions' not in relationships
+        assert 'registrations' not in relationships
+        assert 'forked_from' not in relationships
+        parent_link = relationships['parent']['links']['related']['href']
+        assert urlparse(parent_link).path == '/{}sparse/nodes/{}/'.format(API_BASE, parent._id)
+        assert 'sparse' not in relationships['detail']['links']['related']['href']
+        assert 'sparse' in relationships['children']['links']['related']['href']
```

Comment:
worth asserting what the self link is, since you overrode it to be a sparse nodes self link?